Watchman.Discord/Areas/Protection/Controllers/MuteUserController.cs

```diff
@@ -65,34 +65,36 @@ public async Task UnmuteUserAsync(UnmuteCommand command, Contexts contexts)
         [AdminCommand]
         public async Task MutedUsers(MutedUsersCommand mutedUsersCommand, Contexts contexts)
         {
-            var notUnmutedMuteEvents = _mutingHelper.GetNotUnmutedMuteEvents(contexts.Server.Id);
+            var notUnmutedMuteEvents = this._mutingHelper.GetNotUnmutedMuteEvents(contexts.Server.Id);
             var mutedUsers = this._usersService.GetUsersAsync(contexts.Server);
-            var mutedUsersMessageData = await this.GetMuteEmbedMessage(notUnmutedMuteEvents.ToList(), mutedUsers);
+            var messagesService = this._messagesServiceFactory.Create(contexts);
+            var mutedUsersMessageData = this.GetMuteEmbedMessage(notUnmutedMuteEvents.ToList(), mutedUsers);
             if (!mutedUsersMessageData.Values.Any())
             {
-                await this._directMessagesService.TrySendMessage(contexts.User.Id, "Brak wyciszonych użytkowników!");
+                await messagesService.SendResponse(x => x.ThereAreNoMutedUsers());
```

Comment:
czemu wcześniej było jako wiadomość prywatna a teraz już nie jest?

Reply:
Bo wyglądało to na kanale gdzie wysłana była komenda jakby bot nic nie odpowiedział.
A nie ma sensu wysyłać tego samego na kanał i na priv